README.rst

```diff
@@ -3,9 +3,11 @@ Adhocracy customizations for advocate-europe
 
 .. image:: https://api.travis-ci.org/liqd/adhocracy3.mercator.png?branch=master
     :target: http://travis-ci.org/liqd/adhocracy3.mercator
-.. image:: https://coveralls.io/repos/liqd/adhocracy3.mercator/badge.png?branch=master
+.. image:: https://coveralls.io/repos/liqd/adhocracy3.mercatorz/badge.png?branch=master
```

Comment:
That's a regression too. Looks like some changes by been reverted by rebasing. I'm afraid I cannot merge this as is :(